src/Dialog/DialogBottomSheet.test.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import {render} from '@testing-library/react'
+import {Dialog} from './Dialog'
+import MatchMediaMock from 'jest-matchmedia-mock'
+import {behavesAsComponent, checkExports, renderStyles, checkStoriesForAxeViolations} from '../utils/testing'
+import {mediaQueries} from '../utils/layout'
+
+let matchMedia: MatchMediaMock
+
+describe('Dialog', () => {
+  beforeAll(() => {
+    matchMedia = new MatchMediaMock()
+  })
+
+  afterAll(() => {
+    matchMedia.clear()
+  })
+
+  behavesAsComponent({
+    Component: Dialog,
+    options: {skipAs: true, skipSx: true},
+    toRender: () => (
+      <Dialog onClose={() => {}}>
+        <div>Hidden when narrow</div>
+      </Dialog>
+    ),
+  })
+
+  checkExports('Dialog', {
+    default: Dialog,
```

Comment:
I don't think this has a default export.

```suggestion
    default: undefined,
```